app/controllers/profile_controller.rb

```diff
@@ -8,4 +8,29 @@ def index
     @cooked_masterpieces = current_user.cooked_masterpieces
     @liked_masterpieces = current_user.liked_masterpieces
   end
+
```

Comment:
no need for before_filter :authenticate_user! because now we don't need user to sign in to see this page